Tools/jit/_llvm.py

```diff
@@ -69,6 +69,11 @@ async def _get_brew_llvm_prefix(*, echo: bool = False) -> str | None:
 
 @_async_cache
 async def _find_tool(tool: str, *, echo: bool = False) -> str | None:
+    # Explicitly defined LLVM installation location
+    if (llvm_tools_dir := os.getenv("LLVM_TOOLS_INSTALL_DIR")) is not None:
```

Comment:
Should we document this somewhere? If it gets a NEWS entry, then we probably have (or need) a list of environment variables that may need to be set when building.

I'd expect the list to be in the devguide, really, which is a different repo. But do we have one in the main repo?

Reply:
Considering the JIT is still experimental, I don't think we should document this in the CPython docs. I agree that it should be in the devguide.

Reply:
We have docs for building in https://github.com/python/cpython/blob/main/Tools/jit/README.md#building. We'd probably want to update this for now.